src/Impl/RestPublisher.php

```diff
@@ -30,12 +30,12 @@ public function __construct(
     ) {
         $this->headers = $this->buildHttpHeaders($authToken);
         $this->messageIngestionEndpointUri = $this->buildMessageIngestionUri($ingestionEndpointUri, $channel);
-        $this->payloadTypeName = $this->convertToPayloadTypeName($channelSchemaName);
+        $this->payloadTypeName = self::convertToPayloadTypeName($channelSchemaName);
         $this->httpRequester = $httpRequester;
         $this->jsonProvider = $jsonProvider;
     }
 
-    private function convertToPayloadTypeName($channelSchemaName): string
+    public static function convertToPayloadTypeName($channelSchemaName): string
```

Comment:
Make open, because the caller doesn't have another way to inject schema name node to wrap message payload.

Idea: implement a method in MessageBuilder, that will accept the schema and do the wrapping of payload; FYI @marekczajkowski and @pawel-boguski-ds